### PR TITLE
perf: render in viewrange not all model 

### PR DESCRIPTION
**Pull request by lumixraku**
close https://github.com/dream-num/univer-pro/issues/2716

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

**Comment by DR-Univer** [comment]
image
这个报告 能不能算一个fps或者frame耗时的平均值，直接显示出来。不用点击里面一个个的看

File: packages/engine-render/src/components/sheets/constants.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const BORDER_Z_INDEX = 50;
+export const FONT_EXTENSION_Z_INDEX = 45;
+export const BG_Z_INDEX = 21;
+export const PRINTING_BG_Z_INDEX = 21;
+
+export const EXPAND_SIZE_FOR_RENDER_OVERFLOW = 20;
```
Comment (VicKun4937) [inline packages/engine-render/src/components/sheets/constants.ts:22]: should we provide a function or poperty fot user to customize the overfollow testing scope

Comment (lumixraku) [reply packages/engine-render/src/components/sheets/constants.ts:22]: customize in facade?

Comment (wzhudev) [reply packages/engine-render/src/components/sheets/constants.ts:22]: Add it to config service.

**Review by wzhudev — APPROVED** [review]

**Pull request merged**